src/main/java/messenger/ImageMessengerRequestProcessor.java

```diff
@@ -0,0 +1,55 @@
+package messenger;
+
+import notification.Notification;
+import request.RequestProcessor;
+
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.MultipartHttpServletRequest;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+/**
+ * Created by annterina on 11.11.17.
+ */
+public class ImageMessengerRequestProcessor extends RequestProcessor {
+
+    MultipartFile image;
```

Comment:
What happened to access modifiers? Are they package-private on purpose?

Reply:
If they are not changed from outside they should be private

Reply:
Done!